avm/res/network/network-manager/security-admin-configuration/rule-collection/rule/main.bicep

```
@@ -58,10 +58,10 @@ param priority int
 param protocol string
 
 @sys.description('Optional. List of destination port ranges. This specifies on which ports traffic will be allowed or denied by this rule. Provide an (*) to allow traffic on any port. Port ranges are between 1-65535.')
-param sourcePortRanges array = []
+param sourcePortRanges sourcePortRangesType?
 
 @sys.description('Optional. The source filter can be an IP Address or a service tag. Each filter contains the properties AddressPrefixType (IPPrefix or ServiceTag) and AddressPrefix (using CIDR notation (e.g. 192.168.99.0/24 or 2001:1234::/64) or a service tag (e.g. AppService.WestEurope)). Combining CIDR and Service tags in one rule filter is not permitted.')
-param sources array = []
+param sources sourcesType?
```

```suggestion
param sources sourcesType
```

Type is already nullable